tests/qiskit_convert_test.py

```diff
@@ -961,3 +961,42 @@ def test_unitary_gate() -> None:
     assert cmds[0].op.type == OpType.Unitary1qBox
     assert cmds[1].op.type == OpType.Unitary2qBox
     assert cmds[2].op.type == OpType.Unitary3qBox
+
+
+def test_ccz_conversion() -> None:
+    qc_ccz = QuantumCircuit(4)
+    qc_ccz.append(qiskit_gates.CCZGate(), [0, 1, 2])
+    qc_ccz.append(qiskit_gates.CCZGate(), [3, 1, 0])
+    tkc_ccz = qiskit_to_tk(qc_ccz)
+    assert tkc_ccz.n_gates_of_type(OpType.CnZ) == tkc_ccz.n_gates == 2
```

Comment:
Do you want to do any additional checks to find out if the qubits are set in the right way?

Reply:
Do you mean check the unitary of the converted circuit to ensure that the conversion is not putting the control and target qubits in the wrong place?

Reply:
Yes

Reply:
Done, let me know if theres anything else